src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -229,6 +231,12 @@ export const useEstimateTransactionGas = ({
           safeTxGas,
           approvalAndExecution,
         })
+
+        // TODO: Find a proper fix.
```

Comment:
why it cant be considered a proper fix?

Reply:
it was reimplemented, take a look if you can